pandas/_libs/parsers.pyx

```diff
@@ -229,9 +229,9 @@ cdef extern from "parser/tokenizer.h":
         int64_t skip_first_N_rows
         int64_t skipfooter
         # pick one, depending on whether the converter requires GIL
-        float64_t (*double_converter)(const char *, char **,
-                                      char, char, char,
-                                      int, int *, int *) nogil
+        double (*double_converter)(const char *, char **,
```

Comment:
This function actually returns a double; not sure why Cython was OK with the previous declaration but had to explictly change it as part of this PR for things to cythonize